src/helpers/EventHelper.ts

```diff
@@ -40,10 +40,23 @@ export default class EventHelper {
 
   static async _onSubscriptionChanged(newSubscriptionState: boolean | undefined) {
     EventHelper.onSubscriptionChanged_showWelcomeNotification(newSubscriptionState);
+    EventHelper.onSubscriptionChanged_sendCategorySlidedownTags(newSubscriptionState);
     EventHelper.onSubscriptionChanged_evaluateNotifyButtonDisplayPredicate();
     EventHelper.onSubscriptionChanged_updateCustomLink();
   }
 
+  private static async onSubscriptionChanged_sendCategorySlidedownTags(isSubscribed: boolean | undefined) {
+    if (isSubscribed === true) {
+        const promptOptions = await OneSignal.context.appConfig.userConfig.promptOptions;
+        const isUsingCategoryConfig = !!promptOptions && !!promptOptions.slidedown &&
```

Comment:
Should check for tag length like below.

Also should be a shared method so this doesn't have to be duplicated code.

Reply:
Done.